packages/@o3r/design/builders/generate-style/schema.json

```diff
@@ -0,0 +1,111 @@
+      "enum": [
+        "css",
+        "scss",
+        "sass"
+      ],
+      "description": "Type of the variables to generate for a Design Token."
+    },
+    "variableReferenceType": {
+      "type": "string",
+      "enum": [
+        "css",
+        "scss",
+        "sass"
+      ],
+      "description": "Type of the variables to generate for a Design Token reference."
+    },
+    "designTokenFilePatterns": {
+      "description": "Path patterns to the Design Token JSON files (it supports Node dependency paths).",
+      "oneOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      ]
+    },
+    "output": {
+      "type": "string",
+      "description": "Path to generate the metadata for CMS. The Metadata will be generated only if the file path is specified"
+    },
+    "defaultStyleFile": {
+      "type": "string",
+      "default": "src/theme.scss",
+      "description": "File path to generate the variable if not determined by the specification"
+    },
+    "metadataOutput": {
+      "type": "string",
+      "description": "If specified, all the generated CSS variable will be generated in the given file. Otherwise, the output file will be determined based on the Variable parameters."
+    },
+    "metadataIgnorePrivate": {
+      "type": "boolean",
+      "default": false,
+      "description": "Ignore the private variable in the metadata generation."
+    },
+    "rootPath": {
+      "type": "string",
+      "description": "Root path used to determine the CSS file to edit if specified by the o3rTargetFile token extension. It will default to the Design Token file folder."
+    },
+    "watch": {
+      "type": "boolean",
+      "default": false,
+      "description": "Enable Watch mode"
+    },
+    "failOnDuplicate": {
+      "type": "boolean",
+      "default": false,
+      "description": "Determine if the process should stop in case of Token duplication"
+    },
+    "templateFile": {
+      "description": "Path to a template file(s) to apply as default configuration to a Design Token extension. In case of multiple files, the JSON will be deeply merged in the order defined by the list.",
+      "oneOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      ]
+    },
+    "prefix": {
+      "type": "string",
+      "description": "Prefix to append to generated variables"
+    },
+    "prefixPrivate": {
+      "type": "string",
+      "description": "Prefix to append to generated private variables"
+    },
+    "renderPrivateVariableTo": {
+      "type": "string",
+      "enum": [
+        "scss",
+        "sass"
+      ],
+      "description": "Generate the Private Variable to the given language (if not specified, in case of CSS, the variable will be ignored in generated style)"
```

Comment:
```suggestion
      "description": "Generate the Private Variable in the given language (if not specified, in case of CSS, the variable will be ignored in the generated style)"
```